dynawo/sources/Common/Dictionaries/validateDictionaries.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
It is recommended to have no shebang at all if we do not provide any virtual environment to the users and if we target "old" environments:
https://www.python.org/dev/peps/pep-0394/#recommendation

Reply:
I will remove them all